tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -763,6 +763,24 @@ def test_join_identity(dut):
     clk_gen.kill()
 
 
+@cocotb.test()
+def test_edge_identity(dut):
+    """
+    Test that Edge triggers returns the same object each time
+    """
```

Comment:
This test passed before this patch too, but I figured it was worth adding anyway